Sources/SwiftUINavigation/Documentation.docc/Articles/DestructuringViews.md

```diff
@@ -117,7 +117,7 @@ The "pattern" for the ``IfCaseLet`` is expressed by what is known as a "[case pa
 A case path is like a key path, except it is specifically tuned for abstracting over the
 shape of enums rather than structs. A key path abstractly bundles up the functionality of getting 
 and setting a property on a struct, whereas a case path bundles up the functionality of "extracting"
-a value from an enum and "embedding" a value into an enum. They are an indispensible tool for 
+a value from an enum and "embedding" a value into an enum. They are indispensable tools for 
```

Comment:
Since I think we want to focus on the singular concept of "case path," let's keep the "an," but otherwise good catch :smile:

```suggestion
a value from an enum and "embedding" a value into an enum. They are an indispensable tools for 
```